Documentation/install/system_requirements.rst

```diff
@@ -289,6 +289,19 @@ ICMP 8/0                 egress          ``worker-sg`` (self) health checks
           Routing mode, you can condense all rules into ingress/egress ANY
           port/protocol to/from self.
 
+The following ports should also be available on each node:
+
+======================== ==========  ==========================================
+Port Range / Protocol    Interface   Description
```

Comment:
I found the `interface` title here and the classification in the rows confusing. I'm not sure what I would do as a user to act upon this information.

Maybe it would be more clear to omit this column entirely?

Reply:
I see your point (also your comment above) and think I'll drop this column for now.